src/main/java/io/github/aquerr/chestrefill/entities/RefillableContainer.java

```diff
@@ -434,6 +446,10 @@ public Builder itemProvider(ItemProvider itemProvider)
             return this;
         }
 
+        public Builder(boolean indestructible) {
```

Comment:
Co to jest? Tu chyba miała być metoda ustawiająca pole i zwracająca Builder a nie konstrutkor? 😄